pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -66,13 +67,14 @@ const (
 	prepareAppNameFmt = "rook-ceph-osd-prepare-%s"
 	osdAppNameFmt     = "rook-ceph-osd-%d"
 	// OsdIdLabelKey is the OSD label key
-	OsdIdLabelKey                  = "ceph-osd-id"
-	serviceAccountName             = "rook-ceph-osd"
-	portableKey                    = "portable"
-	cephOsdPodMinimumMemory uint64 = 2048 // minimum amount of memory in MB to run the pod
-	bluestorePVCMetadata           = "metadata"
-	bluestorePVCWal                = "wal"
-	bluestorePVCData               = "data"
+	OsdIdLabelKey                          = "ceph-osd-id"
```

Comment:
Nothing changed in this block? Strange that the linter changed the spacing like this...